tests/ui_tests/test_admin_page.py

```diff
@@ -347,4 +352,73 @@ def test_delete_group_from_groups_page(self, app_fs):
+        with allure.step('Check that group has been deleted'):
+            assert len(page.table.get_all_rows()) == 0, "There should be 0 groups"
+
+
+class TestAdminPolicyPage:
+    """Tests for the /admin/policies"""
+
+    custom_policy = AdminPolicyInfo(
+        name="Test policy name",
+        description="Test policy description",
+        role="ADCM User",
+        users="admin,status",
+        groups=None,
+    )
+
+    def test_open_by_tab_admin_policies_page(self, app_fs):
+        """Test open /admin/policies from left menu"""
+
+        intro_page = AdminIntroPage(app_fs.driver, app_fs.adcm.url).open()
+        policies_page = intro_page.open_policies_menu()
+        policies_page.wait_page_is_opened()
+        policies_page.check_all_elements()
+        policies_page.check_admin_toolbar()
+
+    def test_create_policy_on_admin_groups_page(self, app_fs):
+        """Test create a group on /admin/policies"""
+
+        policies_page = AdminPoliciesPage(app_fs.driver, app_fs.adcm.url).open()
+        policies_page.create_policy(
+            self.custom_policy.name,
+            self.custom_policy.description,
+            self.custom_policy.role,
+            self.custom_policy.users,
+            self.custom_policy.groups,
+        )
+        current_policies = policies_page.get_all_policies()
+        with allure.step('Check that there are 2 policies'):
+            assert len(current_policies) == 2, "There should be 2 policies on the page"
+            assert current_policies == ["default", self.custom_policy.name], "Created policy should be on the page"
+
+    def test_check_pagination_policy_list_page(self, app_fs):
+        """Test pagination on /admin/policies page"""
+
+        policies_page = AdminPoliciesPage(app_fs.driver, app_fs.adcm.url).open()
+        with allure.step("Create 10 policies"):
+            for i in range(10):
+                policies_page.create_policy(
+                    f"{self.custom_policy.name}_{i}",
+                    self.custom_policy.description,
+                    self.custom_policy.role,
+                    self.custom_policy.users,
+                    self.custom_policy.groups,
+                )
+        policies_page.table.check_pagination(second_page_item_amount=1)
+
+    def test_delete_policy_from_policies_page(self, app_fs):
+        """Test delete custom group on /admin/policies page"""
+
+        policies_page = AdminPoliciesPage(app_fs.driver, app_fs.adcm.url).open()
+        policies_page.create_policy(
+            self.custom_policy.name,
+            self.custom_policy.description,
+            self.custom_policy.role,
+            self.custom_policy.users,
+            self.custom_policy.groups,
+        )
+        policies_page.select_all_groups()
+        policies_page.click_delete_button()
+        with allure.step('Check that policy has been deleted'):
+            assert len(policies_page.table.get_all_rows()) == 1, "There should be 1 default groups"
```

Comment:
I've got an impression it is about policies, so I guess it's typos with "select_all_groups" and "There should be 1 default group" :smile_cat:

Reply:
changed, thnx!